src/features/public/Donations/styles/ThankYou.module.scss

```diff
@@ -60,7 +60,7 @@
   justify-content: center;
 }
 .thankyouImage {
-  background-image: url('/tenants/planet/images/donations/donationComplete.jpg');
+  background-image: url('/tenants/planet/images/donations/donationComplete_1.jpg');
```

Comment:
Please replace the existing image. [Cleanup]

Reply:
Done